exotic.py

```diff
@@ -972,11 +972,15 @@ def realTimeReduce(i):
 
     for imageFile in timeSortedNames:
 
-        hDul = fits.open(imageFile)  # opens the fits file
+        hDul = fits.open(name=imageFile, memmap=False, cache=False, lazy_load_hdus=False)  # opens the fits file
         # Extracts data from the image file and puts it in a 2D numpy array: imageData
         imageData = fits.getdata(imageFile, ext=0)
         header = fits.getheader(imageFile)
 
+        hdul.close()
+        del hdul[0].data
```

Comment:
Why no `del(hdul)` to delete the data and the header?

Reply:
Good point. Changing to `del hdul` instead.

Reply:
Note the case of the variables. ... `hDul` versus `hdul` in the `del` statement. There won't be a need for `hdul = None` if `del hdul` is used. 

On another note, the same issue is apparent in several other areas in the code. where `fits.open(...)` is utilized.